tasks/deploy.yaml

```
@@ -0,0 +1,28 @@
+variables:
+  - name: ARCH
+    description: "What architecture to use"
+    default: "amd64"
+  - name: DEPLOY_IMAGE
+    description: "Container image to use to run uds deploy in"
+    default: "ghcr.io/defenseunicorns/build-harness/build-harness:2.0.28"
+  - name: BUNDLE_VERSION
+    description: "Version of the bundle to deploy"
+    default: "0.2.16"
```

```suggestion
    # x-release-please-start-version
    default: "0.2.16"
    # x-release-please-end
```

Do we need to add release please versioning to this default? If so we will also need to add this file to release-please-config.json in this project

---

I think this is still a little WIP (?)
But I added a TODO comment just in case..